configs/dlstreamer/framework-pipelines/yolov5_pipeline/yolov5s_effnetb0.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+#
+# Copyright (C) 2023 Intel Corporation.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+if [ "$INPUTSRC_TYPE" == "REALSENSE" ]; then
+        decode_pp="! videoconvert ! video/x-raw,format=BGR"
+	# TODO: update with vaapipostproc when MJPEG codec is supported.
+	echo "Not supported until D436 with MJPEG." > /tmp/results/pipeline$cid_count.log
+	exit 0
+fi
+
+# arc only
+# GST_VAAPI_DRM_DEVICE=/dev/dri/renderD129
+# export GST_VAAPI_DRM_DEVICE="$GST_VAAPI_DRM_DEVICE"
+
+DEVICE="CPU" #GPU|CPU|MULTI:GPU,CPU
+PRE_PROCESS="pre-process-backend=vaapi-surface-sharing pre-process-config=VAAPI_FAST_SCALE_LOAD_FACTOR=1 batch-size=1 nireq=4 gpu-throughput-streams=4" #""|pre-process-backend=vaapi-surface-sharing pre-process-config=VAAPI_FAST_SCALE_LOAD_FACTOR=1 batch-size=1 nireq=4 gpu-throughput-streams=4 
+PRE_PROCESS2="pre-process-backend=vaapi-surface-sharing pre-process-config=VAAPI_FAST_SCALE_LOAD_FACTOR=1 reclassify-interval=1 batch-size=1 nireq=4 gpu-throughput-streams=4" #""|pre-process-backend=vaapi-surface-sharing pre-process-config=VAAPI_FAST_SCALE_LOAD_FACTOR=1 reclassify-interval=1 batch-size=1 nireq=4 gpu-throughput-streams=4
```

Comment:
what are pre-process and pre_process2?  please use more descriptive name to indicate the cases